org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/parser/antlr/XtextAntlrGeneratorFragment2.xtend

```diff
@@ -390,31 +390,31 @@ class XtextAntlrGeneratorFragment2 extends AbstractAntlrGeneratorFragment2 {
 		'''
 			@�Singleton�
 			public static final class NameMappings {
-				
+
 				�IF partitions.size > 1�
 					�FOR partition : partitions.indexed�
 						private static final class Init�partition.key� {
-							private static void doInit(�ImmutableMap�.Builder<�AbstractElement�, �String�> builder, �grammar.grammarAccess� grammarAccess) {
+							private static void doInit(�Map�<�AbstractElement�, �String�> builder, �grammar.grammarAccess� grammarAccess) {
```

Comment:
I'd call the parameter `map` now

Reply:
I left it as it is since the passed map is still some kind of a 'builder' since it is just used to build the final immutable copy. And it required less charges. 
But if you prefer just `map`, I'll change that.

Reply:
@HannesWell I see your point, but `builder` sounds strange to me; let's wait for @szarnekow 's review anyway.

Reply:
Understand. What about naming it about the content? So just `mapping`?
In general repeating the type in the variable name is often superfluous from a Clean-Code point of view.

Reply:
`mappings` sounds fine to me

Reply:
@szarnekow are you fine as well with `mappings`?

Reply:
Yes, that’s a good fit